src/history.rs

```diff
@@ -762,10 +744,10 @@ mod test {
 
         // One more to get the remainder
         let bytes_written = h.report(&mut report_dest).unwrap().unwrap();
-        assert_eq!(bytes_written.get(), 59);
+        assert_eq!(bytes_written.get(), 51);
         let log_report = WireReport::new(&report_dest[..bytes_written.get()]).unwrap();
         assert_eq!(log_report.n_clocks() as usize, h.clocks.len());
-        assert_eq!(log_report.n_log_entries(), 6);
+        assert_eq!(log_report.n_log_entries(), 4);
```

Comment:
Why did this change?

Reply:
The history struct is larger than it used to be, causing the log slice to be decreased by 2 entries in this test, meaning there are 2 less entries written at the start of the test.